modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/TxCommands.java

```diff
@@ -138,8 +139,12 @@ else if (args.getOperation() == VisorTxOperation.KILL)
                     ", consistentId=" + key.consistentId() +
                     "]");
 
-                for (VisorTxInfo info : entry.getValue().getInfos())
-                    logger.info(info.toUserString());
+                for (VisorTxInfo info : entry.getValue().getInfos()) {
+                    if (Objects.equals(info.getXid(), info.getNearXid()))
+                        logger.info(info.toUserString());
+                    else
+                        logger.info(info.toUserString() + " [Near node may be missed]");
+                }
```

Comment:
Also looks like that a remark about missing node is correct only if the mode is non-verbose. In verbose mode we don't remove transactions that meet this condition.

Reply:
This is true. I suggest to collect only near transactions if non verbose mode.